apitools/base/py/base_api.py

```diff
@@ -376,6 +378,15 @@ def num_retries(self, value):
         self.__num_retries = value
 
     @property
+    def check_response_func(self):
```

Comment:
Looks like we don't really need this property? It is only being used privately except in the test? In the test it can be passed through the constructor anyway?